docs/examples/fl_experiment_tracking_mlflow.rst

```diff
@@ -0,0 +1,150 @@
+``learner_with_mlflow.py`` (under NVFlare/examples/advanced/experiment-tracking/pt) contains the code written for the MLflowWriter syntax.
+
+The :class:`MLflowWriter<nvflare.app_opt.tracking.mlflow.mlflow_writer.MLflowWriter>` mimics the syntax of mlflow, to make it easier to use existing code
+that is using MLflow for metrics tracking. Instead of writing to the MLflow tracking server, however, the MLflowWriter creates and sends an event
+within NVFlare with the information to track.
+
+Finally, :class:`ConvertToFedEvent<nvflare.app_common.widgets.convert_to_fed_event.ConvertToFedEvent>` converts local events to federated events.
+This changes the event ``analytix_log_stats`` into a fed event ``fed.analytix_log_stats``, which will then be streamed from the clients to the server.
+
+.. literalinclude:: ../../examples/advanced/experiment-tracking/mlflow/jobs/hello-pt-mlflow/app/config/config_fed_server.json
+   :language: json
+   :linenos:
+   :caption: config_fed_server.json
+
+Under the component section in the server config, we have the
+:class:`MLflowReceiver<nvflare.app_opt.tracking.mlflow.mlflow_receiver.MLflowReceiver>`
+of type :class:`AnalyticsReceiver<nvflare.app_common.widgets.streaming.AnalyticsReceiver>`.
+
+This component receives events from the clients and internally buffers them before writing
+to the MLflow tracking server. The default "buffer_flush_time" is one second, but this can be
+configured as an arg in the component config for MLflowReceiver.
+
+Notice how the accepted event type ``"fed.analytix_log_stats"`` matches the output of
+:class:`ConvertToFedEvent<nvflare.app_common.widgets.convert_to_fed_event.ConvertToFedEvent>` in the client config.
+
+
+Adding MLflow logging to your Code
+-------------------------------------------
+
+In this exercise, all of the MLflow code additions will be made in ``learner_with_mlflow.py``.
+
+First we must initialize our MLflow writer we defined in the client config:
+
+.. literalinclude:: ../../examples/advanced/experiment-tracking/pt/learner_with_mlflow.py
+   :language: python
+   :lines: 102-105
+   :lineno-start: 102
+   :linenos:
+
+The ``LearnerExecutor`` passes in the component dictionary into the ``parts`` parameter of ``initialize()``.
+We can then access the ``MLflowWriter`` component we defined in ``config_fed_client.json``
+by using the ``self.analytic_sender_id`` as the key in the ``parts`` dictionary.
+Note that ``self.analytic_sender_id`` defaults to ``"analytic_sender"``,
+but we can also define it in the client config to be passed into the constructor.
+
+Now that our writer is set to ``MLflowWriter``,
+we can write and stream training metrics to the server in ``local_train()``:
+
+.. literalinclude:: ../../examples/advanced/experiment-tracking/tensorboard/jobs/tensorboard-streaming/app/custom/pt_learner.py
+   :language: python
+   :lines: 148-182
+   :lineno-start: 148
+   :linenos:
```

Comment:
This PT Learner should reference to mlflow job instead of tensorboard job?

Reply:
Thanks, corrected this